spec/support/helpers.rb

```diff
@@ -5,4 +5,11 @@ def sign_in(user)
     fill_in 'Password', with: user.password
     click_button 'Log in'
   end
+
+  def add_review(signed_in_user, program, review)
```

Comment:
Unused method argument - `signed_in_user`. If it's necessary, use `_` or `_signed_in_user` as an argument name to indicate that it won't be used.